sdk/python/feast/offline_server.py

```diff
@@ -215,9 +217,14 @@ def offline_write_batch(self, command: dict, key: str):
         )
 
         assert len(feature_views) == 1, "incorrect feature view"
+        feature_view = feature_views[0]
+        assert_permissions(
+            resources=feature_view, actions=[AuthzedAction.WRITE_OFFLINE]
```

Comment:
maybe it's `[AuthzedAction.WRITE, AuthzedAction.WRITE_OFFLINE]`? Since `WRITE` is also including `WRITE_OFFLINE`